docker-compose.yml

```
@@ -87,20 +87,19 @@ services:
   gateway:
     container_name: gateway
     image: docker.io/qiskit/quantum-serverless-gateway:0.0.6
-    command: python manage.py runserver 0.0.0.0:8000
+    command: gunicorn main.wsgi:application --bind 0.0.0.0:8000
```

```suggestion
    command: gunicorn main.wsgi:application --bind 0.0.0.0:8000  --workers=4
```

Gunicorn needs workers in order to handle multiple requests. By default worker is 1 and it can handle only 1 requests in a single thread therefore it was not able to call other url in the same app